librz/analysis/p/analysis_cil.c

```diff
@@ -0,0 +1,37 @@
+// SPDX-FileCopyrightText: 2022 wingdeans
```

Comment:
If it's possible, please use the real name and add an email for future contact, if there will be the need.